aidermacs.el

```diff
@@ -163,7 +163,8 @@ PROMPT is the text to display.  INITIAL-INPUT is the default value."
    ["Drop Files"
     ("j" "Drop File" aidermacs-drop-file)
     ("J" "Drop Current File" aidermacs-drop-current-file)
-    ("k" "Drop All Files" aidermacs-drop-all-files)]
+    ("K" "Drop All Files" aidermacs-drop-all-files)
+    ("k" "Drop From Dired (marked)" aidermacs-batch-drop-dired-marked-files)]
```

Comment:
I changed this because `K` is more destructive, so I figured it should be the harder one to press. If you don't like to change an existing key then I can reverse them.

Reply:
I like it!

Reply:
Maybe we should rename it to `Drop Using Dired (marked)`

Reply:
Add From Dired is expecting the user to be already in an dired buffer, but drop-dired will open a dired buffer for user to batch drop? I haven't had time to test it, is this the expected behavior?

Reply:
No, the idea is you do List Added Files and then from that (or any other Dired buffer) make your selection and then call Drop From Dired. So it's very much the same UI as the adding case.

Reply:
I'm in the middle of refactoring, will test your PR ASAP.